src/components/application_manager/test/application_helper_test.cc

```diff
@@ -237,13 +242,14 @@ TEST_F(ApplicationHelperTest, RecallApplicationData_ExpectHMICleanupRequests) {
   cmd[strings::msg_params][strings::vr_commands][0] = "vrCmd";
   cmd[strings::msg_params][strings::menu_id] = menu_id;
   cmd[strings::msg_params][strings::interaction_choice_set_id] = choice_set_id;
+  smart_objects::SmartObjectSPtr message_to_hmi;
```

Comment:
Please, move this to line before line 252.

Reply:
Done in commit 16f31bfe48381f49e1fee07fb082941f50c21741